commit aa8e114497e28bcd189f55dfe4e6f46816e4d534
Author: Andreas Schneider <asn@cryptomilk.org>
Date:   Tue Jan 14 13:11:07 2014 +0100

    cmake: Fix the FindLibSSH module if no libssh is around.

diff --git a/cmake/modules/FindLibSSH.cmake b/cmake/modules/FindLibSSH.cmake
index 9ea394c..19ed2d0 100644
--- a/cmake/modules/FindLibSSH.cmake
+++ b/cmake/modules/FindLibSSH.cmake
@@ -43,7 +43,7 @@ set(LIBSSH_LIBRARIES
     ${SSH_LIBRARY}
 )
 
-if (LibSSH_FIND_VERSION)
+if (LIBSSH_INCLUDE_DIR AND LibSSH_FIND_VERSION)
   file(STRINGS ${LIBSSH_INCLUDE_DIR}/libssh/libssh.h LIBSSH_VERSION_MAJOR
     REGEX "#define[ ]+LIBSSH_VERSION_MAJOR[ ]+[0-9]+")
 
@@ -63,14 +63,12 @@ if (LibSSH_FIND_VERSION)
     message(STATUS "LIBSSH_VERSION_MAJOR not found in ${LIBSSH_INCLUDE_DIR}/libssh/libssh.h, assuming libssh is too old")
     set(LIBSSH_FOUND FALSE)
   endif (LIBSSH_VERSION_MAJOR)
-endif (LibSSH_FIND_VERSION)
+endif (LIBSSH_INCLUDE_DIR AND LibSSH_FIND_VERSION)
 
 # If the version is too old, but libs and includes are set,
 # find_package_handle_standard_args will set LIBSSH_FOUND to TRUE again,
 # so we need this if() here.
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(LibSSH DEFAULT_MSG LIBSSH_LIBRARIES LIBSSH_INCLUDE_DIR)
-
 find_package_handle_standard_args(LibSSH
                                   FOUND_VAR
                                     LIBSSH_FOUND
